packages/vaadin-grid/test/visual/lumo/grid.test.js

```diff
@@ -219,6 +219,36 @@ describe('grid', () => {
           await visualDiff(element, `${import.meta.url}_${dir}-sorting-single-desc`);
         });
       });
+
+      describe('row focus', () => {
+        before(async () => {
+          element = fixtureSync(`
+            <vaadin-grid style="width: 550px">
+              <vaadin-grid-column path="name.first" width="200px" flex-shrink="0" frozen></vaadin-grid-column>
+              <vaadin-grid-column path="name.last" width="200px" flex-shrink="0"></vaadin-grid-column>
+              <vaadin-grid-column path="location.city" width="200px" flex-shrink="0"></vaadin-grid-column>
+            </vaadin-grid>
+          `);
+          element.items = users;
+          flushGrid(element);
+
+          // Scroll all the way to end
+          element.$.table.scrollLeft = element.__isRTL ? -1000 : 1000;
+          // // Focus a row
+          element.setAttribute('navigating', '');
```

Comment:
**nit:** 

Could the element get focus by a series of `sendKeys({ ... })`? 

Using `sendKeys` covers the case when some default browser styles are unexpectedly applied on top of the theme styles.

Reply:
Good idea, done